code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -4,7 +4,7 @@
 	var/obj/machinery/camera/current = null
 	icon = 'icons/mob/pai.dmi'
 	icon_state = "repairbot"
-	mouse_opacity = 1
+	mouse_opacity = 2

```

Comment:
@AnturK we have this system for this exact reason, so you don't have to do that.